coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/universe/sql/SqlTableCleanup.scala

```diff
@@ -2,10 +2,11 @@ package com.socrata.datacoordinator.truth.universe
 package sql
 
 import java.sql.Connection
+import scala.concurrent.duration.FiniteDuration
 
 import com.rojoma.simplearm.util._
 
-class SqlTableCleanup(conn: Connection, daysDelay: Int = 1) extends TableCleanup {
+class SqlTableCleanup(conn: Connection, daysDelay: FiniteDuration) extends TableCleanup {
   val log = org.slf4j.LoggerFactory.getLogger(classOf[SqlTableCleanup])
   def cleanupPendingDrops(): Boolean = {
     using(conn.createStatement()) { stmt =>
```

Comment:
Yes, that probably won't work.  The SQL is:
WHERE queued_at < now() - ('$daysDelay day' :: INTERVAL)
May be the var should be named just delay and change the sql to delay.toSeconds.  Handle values that are smaller than a second.  BTW, this change is commented out by latter commit/release.